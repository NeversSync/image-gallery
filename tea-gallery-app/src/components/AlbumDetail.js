import React from 'react';
// import PropTypes from 'prop-types';
import Wrapper from './Wrapper';

export default function AlbumDetail(props) {
  return (
    <Wrapper
      images={props.images}
      onDeleteImage={props.onDeleteImage} 
    />
  );
}