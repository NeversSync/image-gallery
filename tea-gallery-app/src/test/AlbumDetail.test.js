import React from 'react';
import renderer from 'react-test-renderer';
import AlbumDetail from '../components/AlbumDetail';

const images = [
  {
    title: 'tea leaves in teapot',
    description: 'what nice leaves',
    url: '/../public/img/teainpot.jpg'
  }
];

const match = {
  params: '123'
};

describe.skip('AlbumDetail', () => {
  it('renders the AlbumDetail view', () => {
    const component = renderer.create(<AlbumDetail match={match} images={images}/>);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });

});