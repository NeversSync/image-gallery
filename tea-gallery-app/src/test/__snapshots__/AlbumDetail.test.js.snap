// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumDetail renders the AlbumDetail view 1`] = `<br />`;
